xla/service/hlo_creation_utils.h

```diff
@@ -22,6 +22,7 @@ limitations under the License.
 #include <vector>
 
 #include "absl/types/span.h"
+#include "xla/hlo/builder/xla_computation.h"
```

Comment:
You need to add a corresponding BUILD dependency to the hlo_creation_utils target.

Reply:
Fixed by replacing the include with "xla/client/xla_computation.h"
I guess we should remove this alias at some point.

Reply:
Updated, there was just one file; the rest just depend on ":hlo_creation_utils".